### DISCUSSION TITLE
Welcome to react-admin Discussions!

### DISCUSSION THREAD

**Discussion by MFinnnne** [MFinnnne/react-admin#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chaoStart** [comment]
这个数据库的数据的代码也在里面吗？我跟着上硅谷的课程，没有返回商品的数据，你知道怎么做的吗？谢谢大佬
